app/templates/search_organisation.hbs

```diff
@@ -37,20 +37,22 @@
         {{#if filteredResults}}
           <ul class="list list-activity">
             {{#each filteredResults as |organisation|}}
-              <li class="org_search">
-                <div class="row" {{action "selectOrganisation" organisation}}>
-                  <div class="small-12 large-12 columns">
-                    {{organisation.nameEn}}
+              {{#link-to "account_details" (query-params orgId=organisation.id)}}
```

Comment:
This redirects to account_details, but is there any scenario where we might need the organisation list in a different navigation flow ?
@Darshan-Chauhan @namrataukirde

Reply:
Right, now there isn't any scenario where organisation list is required. But, I am reverting back 'selectOrganisation' action. And will add `backlink` in `cordova-lib` addon.